homeassistant/components/vallox/sensor.py

```diff
@@ -0,0 +1,189 @@
+"""Support for Vallox ventilation unit sensors."""
+
+from datetime import timedelta
+import logging
+
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY, DEVICE_CLASS_TEMPERATURE, TEMP_CELSIUS)
+from homeassistant.helpers.entity import Entity
+
+from . import (
+    DATA_VALLOX_SENSOR_LIST, DOMAIN, MODE_METRIC_KEY, SENSOR_FAN_SPEED,
+    SENSOR_HUMIDITY, SENSOR_REMAINING_TIME_FILTER, SENSOR_TEMP_EXHAUST_AIR,
+    SENSOR_TEMP_EXTRACT_AIR, SENSOR_TEMP_OUTDOOR_AIR, SENSOR_TEMP_SUPPLY_AIR)
+
+_LOGGER = logging.getLogger(__name__)
+
+SENSOR_INSTANCES = {}
+
+SCAN_INTERVAL = timedelta(seconds=60)
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Set up the sensors."""
+    # Abort early if the sensor list was not populated by the vallox domain in
+    # advance.
+    if DATA_VALLOX_SENSOR_LIST not in hass.data:
+        return
+
+    try:
+        name = hass.data[DOMAIN]['name']
+        client = hass.data[DOMAIN]['client']
+
+    except KeyError:
+        _LOGGER.error("Domain data not populated. "
+                      "Check for \"vallox:\" in the configuration.")
+        return
+
+    global SENSOR_INSTANCES
```

Comment:
Don't use global. We can use hass.data to store instances specific to the integration platform.